README.md

```diff
@@ -5,12 +5,13 @@
 3. [Problem](#problem)
 4. [User Journey](#user-journey)
```

Comment:
upload related files in the pull request

Reply:
I removed the console I have no idea how to handle the errors in the front end